dojo/tools/jfrog_xray_api_summary_artifact/parser.py

```diff
@@ -0,0 +1,170 @@
+    def get_scan_types(self):
+        return ["JFrog Xray API Summary Artifact Scan"]
+
+    # This function return a string used to provide some text in the UI (short label)
+    def get_label_for_scan_types(self, scan_type):
+        return scan_type
+
+    # This function return a string used to provide some text in the UI (long description)
+    def get_description_for_scan_types(self, scan_type):
+        return "Import Xray findings in JSON format from the JFrog Xray API Summary/Artifact JSON response"
+
+    # This function return a list of findings
+    def get_findings(self, json_output, test):
+        tree = json.load(json_output)
+        return self.get_items(tree, test)
+
+    def get_items(self, tree, test):
+        items = []
+        if 'artifacts' in tree:
+            artifact_tree = tree['artifacts']
+            for artifactNode in artifact_tree:
+                artifact_general = artifactNode['general']
+                artifact_issues = artifactNode['issues']
+                for node in artifact_issues:
+                    service = decode_service(artifact_general['name'])
+                    item = get_item(node, str(service), test)
+                    items.append(item)
+
+        return items
+
+
+# Retrieve the findings
+def get_item(vulnerability, service, test):
+    cve = "No CVE"
+    cwe = None
+    cvssv3 = None
+    cvss_v3 = "No CVSS v3 score"
+    cvssv3_score = 0.0
+    unique_id_from_tool = None
+    impact_paths = None
+    impact_path = ImpactPath("", "", "")
+
+    if 'severity' in vulnerability:
+        if vulnerability['severity'] == 'Unknown':
+            severity = "Informational"
+        else:
+            severity = vulnerability['severity'].title()
+    else:
+        severity = "Informational"
+
+    # Some entries have no CVE entries, despite they exist. Example CVE-2017-1000502.
+    cves = vulnerability.get('cves', [])
+    if len(cves) > 0:
+        if 'cve' in cves[0]:
+            cve = cves[0]['cve']
+        if len(cves[0].get('cwe', [])) > 0:
+            cwe = decode_cwe_number(cves[0].get('cwe', [])[0])
+        if 'cvss_v3' in cves[0]:
+            cvss_v3 = cves[0]['cvss_v3']
+            cvssv3 = CVSS3.from_rh_vector(cvss_v3).clean_vector()
+            cvssv3_score = CVSS3.from_rh_vector(cvss_v3).base_score
+
+    impact_paths = vulnerability.get('impact_path', [])
+    if len(impact_paths) > 0:
+        impact_path = decode_impact_path(impact_paths[0])
+
+    if 'issue_id' in vulnerability:
+        title = vulnerability['issue_id'] + " - " + impact_path.name + ":" + impact_path.version
+    elif cve:
+        title = str(cve) + " - " + impact_path.name + ":" + impact_path.version
+    else:
+        title = "No CVE - " + impact_path.name + ":" + impact_path.version
+
+    unique_id_from_tool = title
+
+    finding = Finding(
+        service=service,
+        title=title,
+        cwe=cwe,
+        cve=cve,
```

Comment:
I think 'No CVE' can end up here which is an invalid value

Reply:
Initialization of cve chenged from "No CVE" to "none"